tests/python_test.py

```diff
@@ -256,7 +256,7 @@ def test_acquire_release_resource(tmpdir, sanlock_daemon, size, offset):
 
 @pytest.mark.parametrize("align, sector", [
     # Invalid alignment
-    (1024, sanlock.SECTOR_SIZE[0]),
+    (constants.KiB, sanlock.SECTOR_SIZE[0]),
     # Invalid sector size
     (sanlock.ALIGN_SIZE[0], 8192),
```

Comment:
8*constants.KiB

Reply:
8 * constants.KiB is hard to read, 8192 is better. But if we import the units, 8 * KiB is little nicer.